### PR TITLE
apt: no gpg key is installed allows unauthenticated packages

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
@alfredodeza hopefully this is in line with what you recommended during the discussions of https://github.com/ceph/ceph-deploy/pull/362 and https://github.com/ceph/ceph-deploy/pull/363. What do you think ?

**Comment by alfredodeza** [comment]
@dachary we need to update the docs on contributing, but ceph-deploy has been (for a little while) using ticket identifiers on commit messages. This is prepared by a prepare-commit-msg script inside the `.git/hooks/` dir. The script is this https://gist.github.com/alfredodeza/6d62d99a95c9a7975fbe

If you look at pr #355 you will see that the requirement is to have the branch be the same as the ticket. BZ is used for Bugzilla tickets and RM for the redmine tracker.

Would you mind amending the commits to comply with that? (sorry for not having this contributing document ready to point there)

**Comment by alfredodeza** [comment]
This should have a ticket in the tracker as well as in the description (and a link in the issue tracker back to this PR)

**Comment by dachary** [comment]
@alfredodeza I'll do all that. Do you otherwise agree on the patch or does it go in the wrong direction ?

File: ceph_deploy/util/pkg_managers.py [hunk 1]
```diff
@@ -235,10 +235,21 @@ def add_repo_gpg_key(self, url):
         cmd = ['apt-key', 'add', gpg_file]
         self._run(cmd)
 
+    def disable_signatures_check(self):
+        self.remote_conn.logger.info(
+            "Append APT::Get::AllowUnauthenticated 1 to /etc/apt/apt.conf"
+        )
+        self.remote_conn.remote_module.append_to_file(
+            '/etc/apt/apt.conf',
+            'APT::Get::AllowUnauthenticated 1;',
```
Comment (alfredodeza) [inline ceph_deploy/util/pkg_managers.py]: this is a global setting and very unsafe to set it for an install and no set it back. Wouldn't the flag `--allow-unauthenticated` be better as a one time use case?

**Pull request closed without merging**